cmd/goProbe/goProbe.go

```diff
@@ -120,23 +120,19 @@ func main() {
 				logging.LevelFromString(config.Logging.Level) == logging.LevelDebug,
 			),
 			server.WithProfiling(config.API.Profiling),
```

Comment:
Hate to say it, but both `Profiling` and `Metrics` accesses cause a crash again if API is missing in config (because it's nil):
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x11 pc=0xb02496]

goroutine 1 [running]:
main.main()
	/home/fako/Develop/go/src/github.com/els0r/goProbe/cmd/goProbe/goProbe.go:122 +0x936
```
Needs a guard I guess.

Reply:
I'm this close to changing the config handling of goProbe. _This_ close. Not the structure, the handling.